elements/pfe-icon-panel/demo/index.html

```diff
@@ -36,11 +44,11 @@ <h3 slot="header">Stacked &amp; Centered</h3>
       <h3 slot="header">Circled Default</h3>
       Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
     </pfe-icon-panel>
-    <pfe-icon-panel icon="pfe-icon-server-alt2" circled="dark">
+    <pfe-icon-panel icon="pfe-icon-server-alt2" circled color="dark">
```

Comment:
Separated the circled and color values (this helped me implement broadcast values more easily)